app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,146 @@
  *= require_tree .
  *= require_self
  */
+html {
+  background: rgb(0, 22, 47);
+  background: white;
+  width: 100.2vw;
+  height: 100vh;
+  overflow-x: hidden;
+}
+
+#landing {
+  width: 100%;
+  height: 100%;
+  margin-top: 30%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#landing h1 {
+  color: black;
+  font-family: 'proxima-nova', sans-serif;
+  font-size: 4em;
+}
+
+#landing a {
+  display: inline-block;
+  color: black;
+  padding: 10px 20px;
+  background-color: #3498db;
+  text-decoration: none;
+  font-family: 'proxima-nova', sans-serif;
+  font-size: 1.8em;
+}
+
```

Comment:
- [ ] Great job implementing some of the required changes from the previous reviewers. Trust me, you're almost there. It is required to make your mobile app a replica of the design given. You app is working perfectly. What remains is to work on the layout to look almost like the design given. Kudos to your hard work man :handshake: 

### Your Design

![your-app](https://github.com/alexmedinasf/budget_app/assets/26597432/157067b9-c526-443e-a544-644a58ad6c02)


### Temp Design

![temp-design](https://github.com/alexmedinasf/budget_app/assets/26597432/cbc84bd3-c42a-4e82-8b7c-d3366fec6ad4)

Kindly give it quick shot and you would be good to go and enjoy the rest of your weekend :laughing:

Reply:
* Well done with the changes you have made so far but we still have quite a few adjustments the transactions page layout is still different kindly make your buttons cover the width of the display as shown above (in your case the add transaction button and new group buttons should look like the scan button in the design ) 
![image](https://github.com/alexmedinasf/budget_app/assets/52497246/ace5c5d9-24e0-440d-8c85-1124c3b6b6f3)
![image](https://github.com/alexmedinasf/budget_app/assets/52497246/11d73824-2a10-49c0-9ed4-b766ed0e79cb)